packages/ovh-manager/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="common_menu_labs" qtlid="383707">UX Labs</translation>
+   <translation id="common_menu_gamma" qtlid="232401">Sunrise</translation>
+   <translation id="common_menu_v3" qtlid="135931">Ancienne interface client</translation>
+   <translation id="common_menu_portal" translate="none">MyOVH</translation>
+   <translation id="common_menu_partners" qtlid="383512">Partenaires</translation>
+   <translation id="common_menu_billing" qtlid="32950">Facturation</translation>
```

Comment:
Maybe we should have a dependency to billing module to get the same naming ?

Reply:
Should I create an empty `billing` module (with translations only) ?